client/stylesheets/_toCat.scss

```diff
@@ -2094,6 +2110,13 @@ i:hover {
 .helpThree li ul li p {
     margin: 0 0 2px 0;
 }
+
+.helpThree li ul li:last-child {
+    margin-top: 5px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered line-height, list-style-type, margin-top